Public/Src/FrontEnd/Script/Ambients/AmbientGlobal.cs

```diff
@@ -94,6 +96,20 @@ private EvaluationResult AddIfLazy(Context context, ModuleLiteral env, Evaluatio
             return result;
         }
 
+        private EvaluationResult UnsafeOutputFile(Context context, ModuleLiteral env, EvaluationStackFrame args)

```

Comment:
These features should NOT be exposed by default.
Please update this to check if this is enabled in the configuration. (no need to put in commandline, but at least it should be in config.dsc)

```
config({
   frontend: {
        unsafeFileApisAllowed: true
   }
});
```

Reply:
These methods should trhow

---
In reply to: [313650448](https://github.com/microsoft/BuildXL/pull/730#discussion_r313650448) [](ancestors = 313650448)